CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+Catalyst Voter Registration). At the root level, the registration consists of two pieces of data:
+
+1. A Token Registration Payload Object [(TRPO)](#token-registration-payload-object)
+2. A Token Registration Witness array [(TRWA)](#token-registration-witness-array)
+
+### Token Registration Payload Object
+
+The Token Registration Payload Object (TRPO) consists of 4 required fields and then optional, additional fields to
+provide additional context and information as part of the registration payload. For the purposes of this draft CIP and
+in the examples below we will use the top-level JSON Metadata Index of **867** although this could be changed to the
+assigned CIP number if considered for acceptance for the sake of clarity.
+
+### Required Fields
+
+The following fields are required in all token registration submissions.
+
+#### 1. Script Hash (Policy ID)
+
+This should be the hex-encoded policy ID (**Script Hash**) that is being registered. This information is readily 
+available and accessible via Cardano DB-Sync and other chain indexers and is an easy point of reference. The script must
+include at least one key-based signature script requirement in order to witness and sign the policy registration.
+
+If a script requires multiple signatures (multi-sig) then the same validation logic should be applied when verifying
+the authenticity of the registration.
+
+**Example:**
+
+`'00000002df633853f6a47465c9496721d2d5b1291b8398016c0e87ae'`
+
+#### 2. Feature Set
+
+The Feature Set should consist of an array of CIPs that should be applied to the tokens under the given policy. For
+example, a project may specify that their tokens utilize CIP-25 (Cardano NFT Metadata Standard) and CIP-27 (Cardano NFT
+Royalty Standard) or CIP-68, or other, to be developed and determined CIPs.
+
+A registration with an empty feature set, or lack of a specific CIP, should be considered an explicit omission on behalf
+of the project and therefore any potential metadata associated with said CIPs ignored from the time of the registration
+forward.
+
+**Example:**
+
+`[25, 27]`
+
+#### 3. Witness Method
+
+The witness method portion of the payload should describe the method used to generate the witness signature (CIP-8, etc)
+to aid in validation by third parties.
+
+**Example:**
+
+`['CIP-8','Ed25519']`
+
+#### 4. Nonce
+
+As with CIP-36 Voter Registration method, we must include a piece of data that changes to avoid a replay attack. As with
+CIP-36, and stake pool KES counter rotation, the nonce is expected to be an integer value that is incremented as new or
+updated registrations occur. The suggestion to use the current slot tip of the network at the time of submission from 
+CIP-36 is also recommended here.
+
+Validators and indexers observing token registrations should always honor the verified registration with the highest
+nonce value.
+
+**Example:**
+
+`12345`
```

Comment:
Question: can the witness method change between two registrations? If yes, what prevents an attacker from registering with a higher nonce and a completely different witness method that they control?